src/draive/lmm/__init__.py

```
@@ -1,16 +1,24 @@
-from draive.lmm.state import LMM, LMMSession
+from draive.lmm.state import LMM, LMMMemory, RealtimeLMM
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)